ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -330,6 +410,32 @@ Resources:
           commands:
             01_rewrite_apache_conf:
               command: "/bin/bash /opt/sage/bin/apache_conf_rewrite.sh"
+        write_synapse_env:
+          files:
+            /opt/sage/bin/apache_synapse_env.sh:
+              source: "https://raw.githubusercontent.com/Sage-Bionetworks/service-catalog-utils/v1.0.8/linux/opt/sage/bin/apache_synapse_cred_env.sh"
+              mode: "00744"
+              owner: "root"
+              group: "root"
+            /opt/sage/bin/rstudio_synapse_env.sh:
```

Comment:
code might be easier to to follow if you name this the same as the source filename?